### PR TITLE
 [Melodic] add melodic checks

### PR DESCRIPTION
**Pull request by fmessmer**
ref https://github.com/ipa320/care-o-bot/issues/52

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
 install:
-  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci
-
-script: 
-  - .industrial_ci/travis.sh
-  
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```
Comment (mathias-luedtke) [inline .travis.yml]: Why did you change this?

Comment (fmessmer) [reply .travis.yml]: it's the `.travis.yml` we are using in all the other repos...i wanted to harmonize...
is something wrong about the new config?

Comment (fmessmer) [reply .travis.yml]: ah, now i see the `ros-shadow-fixed` being removed....should I re-add?

Comment (fmessmer) [reply .travis.yml]: now it's back to
 - adding melodic jobs
 - removing `sudo` and `dist`
 - adding `service`

I think this is in your interest now

**Comment by fmessmer** [comment]
@ipa-mdl 
any chance this could be made multi-distro compatible?
or do we need a new branch for `melodic`?

**Comment by mathias-luedtke** [comment]
> any chance this could be made multi-distro compatible?
or do we need a new branch for melodic?

`kinetic` and `melodic` support the same (new) API. 
`indigo` is EOL, but some changes have been back-ported.

**Comment by fmessmer** [comment]
superseded by #7
